qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -438,9 +438,8 @@ def test_snap_schedule_with_mgr_restart(self):
         self.fs_snap_schedule_cmd('deactivate', path=testdir, snap_schedule='1M')
 
         new_stats = self.get_snap_stats(testdir)
-        self.assertTrue(new_stats['fs_count'] == new_stats['db_count'])
+        self.assertTrue(new_stats['fs_count'] == new_stats['db_count'] + old_stats['db_count'])
```

Comment:
thanks for taking up this one @vshankar

Reply:
Thank you for the initial backport. It was a tricky one.